xrpl/core/binarycodec/types/currency.py

```diff
@@ -86,7 +86,7 @@ def __init__(self: Currency, buffer: Optional[bytes] = None) -> None:
         if self.buffer[0] != 0:
             # non-standard currency
             self._iso = None
-        elif code_bytes.hex() == "000000":
+        elif self.buffer.hex() == "0" * 40:  # all 0s
```

Comment:
Wow, I didn't know you can multiply a string in python. Haha very cool.

Reply:
It's one of the things that makes it so easy to work with strings in Python :)